logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/MessageSender.kt

```diff
@@ -402,7 +411,11 @@ internal class MessageSenderImpl internal constructor(
                     .handleClientsHaveChangedFailure(failure, conversationId)
                     .flatMap {
                         logger.w("Retrying After Proteus $action Failure: { \"message\" : \"${messageLogString}\"}")
-                        retry()
+                        if (retryCount <= MAX_RETRY_COUNT) {
+                            retry()
+                        } else {
+                            Either.Left(failure)
```

Comment:
suggestion: maybe add a log describing why it didn't retry to avoid any confusion when analysing logs in the future

Reply:
Question: does this add the sliding indicator on the side?